jar-infer/jar-infer-lib/src/test/java/com/uber/nullaway/jarinfer/JarInferTest.java

```diff
@@ -433,6 +436,63 @@ public void jarinferOutputJarIsBytePerByteDeterministic() throws Exception {
     Assert.assertArrayEquals(checksumBytes1, checksumBytes2);
   }
 
+  @Test
+  public void testSignedJars() throws Exception {
+    // Set test configuration paths / options
+    final String baseJarPath = "../test-java-lib-jarinfer/build/libs/test-java-lib-jarinfer.jar";
+    final String pkg = "com.uber.nullaway.jarinfer.toys.unannotated";
+    final String baseJarName = FilenameUtils.getBaseName(baseJarPath);
+    final String workingFolderPath = outputFolder.newFolder("signed_" + pkg).getAbsolutePath();
+    final String inputJarPath = workingFolderPath + "/" + baseJarName + ".jar";
+    final String outputJarPath = workingFolderPath + "/" + baseJarName + "-annotated.jar";
+
+    // Copy Jar to workspace, and sign it
+    Files.copy(
+        Paths.get(baseJarPath), Paths.get(inputJarPath), StandardCopyOption.REPLACE_EXISTING);
+    String ksPath =
+        Thread.currentThread().getContextClassLoader().getResource("testKeyStore.jks").getPath();
+    // JDK 8 only?
```

Comment:
This is kind of annoying...

Reply:
Agreed. I can't think of any other option though.

Reply:
Ok, this is unfortunate, but given CI passes, I vote on leaving it like this for now. We can create a JDK8-only and JDK11-only test targets if this becomes a problem in the future :/

Tool code itself should be version independent.

Reply:
SGTM